databricks/koalas/exceptions.py

```diff
@@ -19,6 +19,14 @@
 """
 
 
+class GroupByError(Exception):
+    pass
+
+
+class DataError(GroupByError):
```

Comment:
Does pandas throw an exception like this?

Reply:
Yep, just followed the pandas'.

https://github.com/pandas-dev/pandas/blob/master/pandas/core/base.py#L124-L125

Reply:
Yes, but seems this behavior has changed. I'll address it.